.github/workflows/ci.yml

```diff
@@ -9,8 +9,8 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-java@v4
         with:
-          distribution: 'temurin'
-          java-version: '11'
+          distribution: temurin
+          java-version: 17
```

Comment:
Is 21 not an option?

Reply:
`21` is an option, but in some cases, compiling w/ higher JDK versions will make consumers also need to bump their versions. My understanding is that as a library, we should keep this as low as feasible.

Granted, we could configure the JVM compile versions in Gradle (probably the best approach), but until we do that, best to keep this low?